### PR TITLE
Use JSCS-preset `node-style-guide`

### PR DESCRIPTION
**Pull request by SimenB**
Ref https://github.com/gulpjs/gulp/issues/904#issuecomment-80772730

`1.12.0` has been released.

It fails on maximum characters in a line. Should I disable that rule, or fix the errors?

### CODE REVIEW

**Comment by callumacrae** [comment]
I usually disable that rule in test directories

**Comment by SimenB** [comment]
JSHint also validates line length, but it's easier to disable it in tests because of the `extend` option which is not available in JSCS (yet, https://github.com/jscs-dev/node-jscs/issues/1106)

File: .jscsrc [hunk 1]
```diff
@@ -1,48 +1,4 @@
...
-  },
-  "requireTrailingComma": {
-      "ignoreSingleLine": true
-  },
-  "requireBlocksOnNewline": true,
-  "validateIndentation": 2,
-  "validateLineBreaks": "LF",
-  "validateQuoteMarks": "'"
+  "preset": "node-style-guide",
+  "maximumLineLength": null
```
Comment (SimenB) [inline .jscsrc]: This validation fails on tests, and is also validated by JSHint. JSHint makes it easy to disable only for tests but kept for all other files

Comment (callumacrae) [inline .jscsrc]: It's not that tricky, just put `// jscs:disable maximumLineLength` at the top of test files

Comment (SimenB) [inline .jscsrc]: It's double validation anyways, and as JSHint is run before JSCS, it will never matter

Comment (phated) [inline .jscsrc]: I'm fine with this

**Pull request merged**